client/src/components/CreateContestForm.js

```diff
@@ -0,0 +1,212 @@
+    'tattoo-6.png', 'tattoo-7.png', 'tattoo-8.png', 'tattoo-9.png', 'tattoo-10.png', 
+    'tattoo-11.png', 'tattoo-12.png']
+
+    return (
+        <form className={classes.formContainer}>
+            <Grid container direction="row">
+                <Grid item xs={12}>
+                    <Typography className={classes.label}>What do you need designed?</Typography>
+                </Grid>
+                <Grid item xs={12}>
+                    <TextField 
+                        id="title"
+                        className={classes.inputMargin}
+                        value={title}
+                        onChange={e => setTitle(e.target.value)}
+                        label="Write a descriptive contest title"
+                        fullWidth
+                        variant="outlined"
+                    />
+                </Grid>
+                <Grid item xs={12}>
+                    <Typography className={classes.label}>Description</Typography>
+                </Grid>
+                <Grid item xs={12}>
+                    <TextField 
+                        id="description"
+                        value={description}
+                        onChange={e => setDescription(e.target.value)}
+                        className={classes.inputMargin}
+                        label="Details about what type of tattoo you want"
+                        fullWidth
+                        multiline
+                        rows={8}
+                        variant="outlined"
+                    />
+                </Grid>
+                <Grid item xs={4}>
+                    <Typography className={classes.label}>Prize Amount</Typography>
+                </Grid>
+                <Grid item xs={8}>
+                    <Typography className={classes.label}>Deadline</Typography>
+                </Grid>
+                <Grid item xs={4}>
+                    <TextField 
+                        id="amount"
+                        value={amount}
+                        error={amountError}
+                        helperText={amountHelperText}
+                        onChange={onAmountChange}
+                        className={classes.inputMargin}
+                        type="number"
+                        InputProps={{
+                            startAdornment: <InputAdornment position="start">$</InputAdornment>,
+                            // endAdornment: <InputAdornment position="end">.00</InputAdornment>,
+                            style: {textAlign: 'right'},
+                        }}
+                        variant="outlined"
+                    />
+                </Grid>
+                <Grid item xs={3}>
+                    <div className={classes.borderedDiv}>
+                        <TextField
+                            id="deadlineDate"
+                            defaultValue="2021-01-12"
+                            inputProps={{ style: { padding: "1rem" } }}
```

Comment:
this input style seems to be used often thorughout this component, can you create a style and pass in a className?

Reply:
It's quite difficult for me to figure out how to make a style with "inputProps" attribute. Do you have any suggestions? Or maybe I'll do it later.